boards/DCM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -96,3 +96,36 @@ void App_SetPeriodicCanSignals_Imu(const struct DcmWorld *world)
         CANMSGS_DCM_NON_CRITICAL_ERRORS_ACCELERATION_Z_OUT_OF_RANGE_UNDERFLOW_CHOICE,
         CANMSGS_DCM_NON_CRITICAL_ERRORS_ACCELERATION_Z_OUT_OF_RANGE_OVERFLOW_CHOICE);
 }
+
+void App_SetPeriodicCanSignals_InverterSwitches(const struct DcmWorld *world)
+{
+    struct InverterSwitches *inverter_switches =
+        App_DcmWorld_GetInverterSwitches(world);
+    struct DcmCanTxInterface *can_tx = App_DcmWorld_GetCanTx(world);
+
+    if (App_InverterSwitches_IsRightOn(inverter_switches))
```

Comment:
can we not just do App_CanTx_SetPeriodicSignal(can_tx, App_InverterIsRightOn) for these? we can make sure that ON_CHOICE maps to true, and OFF_CHOICE maps to false

Reply:
can prob get rid of this fxn after we do what i had suggested above

Reply:
fixed